src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Books from './Books';
+import Addform from './AddForm';
+
+function App() {
+  return (
+    <Router>
+      <nav>
+        <Link className="navLink" to="/">Link#1</Link>
+        <Link className="navLink" to="/">Link#2</Link>
+      </nav>
+      <Books />
+      <Addform />
+      <button type="button">Check status</button>
+      <Routes>
+        <Route path="/" element={<></>} />
+      </Routes>
+    </Router>
```

Comment:
- [ ] It would be better if you can navigate through your app with the links on your app. Currently, your links point to the same page and you only set one route (books page) for the navigation. Please add the route and the link to move to the categories page.

![Capture1](https://user-images.githubusercontent.com/64173189/161108976-b26b67ae-a3cf-4070-aab2-49d7b5c766dc.JPG)

```suggestion
    <Router>
      <nav>
        <Link className="navLink" to="/">Books</Link>
        <Link className="navLink" to="/categories">categories</Link>
      </nav>
      <Books />
      <Addform />
      <button type="button">Check status</button>
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </Router>
```

Remember that the categories must display only a button with text "Check status"


![Capture](https://user-images.githubusercontent.com/64173189/161109644-80567130-f135-4a7d-b7d1-05632c2898db.JPG)